kuttl-test.yaml

```diff
@@ -19,7 +19,7 @@ kind: TestSuite
 reportFormat: JSON
 reportName: kuttl-test-dataplane
 namespace: openstack
-timeout: 600
+timeout: 900
```

Comment:
bumping the timeout so we don't need to change the assertions

Reply:
900 wasn't enough to get rid of the errors: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_dataplane-operator/283/pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl/1682020391662391296